app/src/main/res/layout/activity_backup_picker.xml

```diff
@@ -103,13 +107,26 @@
                 app:checked="@={prefs[`backup-automatic-enabled`]}"
                 app:showIfTrueAnimatedFastOff="@{vm.showAuto}" />
 
+            <CheckBox
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:checked="true"
+                android:dependency="backup-automatic-enabled"
+                android:layout_gravity="center"
+                android:gravity="center"
+                android:layout_marginBottom="20dp"
+                android:text="@string/daily_backup_even_mobile"
+                app:checked="@={prefs[`backup-automatic-mobile`]}"
```

Comment:
I still don't think this will work properly because even though you've changed the logic to default to true, the prefsview I don't think will handle that and will default to false and without a significant rewrite of that component which carries its own risks I don't think its easily solvable. I could take a look at that though as it would be better if we were not constrained by that. For other activities we rely on the XML settings to set the defaults.

For things as they are, the only alternative options are I think you either need to set the setting to true if it isn't currently set so that new users get this defaulted true setting or invert your logic so that it is something like "Back up over WiFi only" which then defaults to false and then you don't need to worry about presetting it to true.

I will take a look at PrefsView and get back to you on whether its easily solvable that way.